test/jdk/tools/jpackage/helpers/jdk/jpackage/test/Executor.java

```diff
@@ -89,6 +89,11 @@ public Executor setRemovePath(boolean value) {
         return this;
     }
 
+    public Executor setTmpDir(String tmp) {
```

Comment:
As this would work only on Windows, I'd throw `IllegalStateException` if the method is called on other platform.